src/validate.js

```diff
@@ -0,0 +1,36 @@
+import { isObject, toErrorList, mergeObjects } from "./utils";
+
+
+function createErrorHandler(formData) {
+  const handler = {
+    __errors: [],
+    addError(message) {
+      this.__errors = [...this.__errors, message];
```

Comment:
Because why not not? :D I must confess I like this syntax more, but push works too. I'm not too attached to it though, so... your call.